zebra/zebra_evpn_mac.c

```diff
@@ -2488,13 +2489,13 @@ int zebra_evpn_mac_gw_macip_add(struct interface *ifp, struct zebra_evpn *zevpn,
 	SET_FLAG(mac->flags, ZEBRA_MAC_AUTO);
 	if (def_gw)
 		SET_FLAG(mac->flags, ZEBRA_MAC_DEF_GW);
+	else
+		SET_FLAG(mac->flags, ZEBRA_MAC_SVI);
```

Comment:
Remove ZEBRA_MAC_SVI set from `zebra_evpn_mac_svi_add` 

```
2551         zebra_evpn_mac_gw_macip_add(ifp, zevpn, NULL, &mac, &macaddr, 0, false);
2552         if (mac)
2553                 SET_FLAG(mac->flags, ZEBRA_MAC_SVI);
```

Reply:
i think his code is already doing that unless I am misreading the diff